src/openforms/api/exception_handling.py

```diff
@@ -26,53 +26,60 @@ def _translate_exceptions(exc):
     return exc
 
 
-def get_validation_errors(validation_errors: dict):
-    for field_name, error_list in validation_errors.items():
-        # nested validation for fields where many=True
-        if isinstance(error_list, list):
-            for i, nested_error_dict in enumerate(error_list):
-                if isinstance(nested_error_dict, dict):
-                    for err in get_validation_errors(nested_error_dict):
-                        err[
-                            "name"
-                        ] = f"{underscore_to_camel(field_name)}.{i}.{err['name']}"
-                        yield err
-
-        # nested validation - recursively call the function
-        if isinstance(error_list, dict):
-            for err in get_validation_errors(error_list):
-                err["name"] = f"{underscore_to_camel(field_name)}.{err['name']}"
+def get_validation_errors(validation_errors: Union[dict, list]):
```

Comment:
these changes are going to cause merge conflicts with https://github.com/open-formulieren/open-forms/pull/1690 which needs backports, so I'll only merge this after the other PR was merged.

Reply:
you should be able to rebase on master now for this